plugins/connection/docker.py

```diff
@@ -115,6 +115,7 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
         # root.
 
         self._docker_args = []
+        self._container_user_cache = {}
```

Comment:
do not cache user as it is not guaranteed to be the same for all calls in the task

Reply:
Do you mean that someone could recreate the container during the playbook/role run?

Reply:
no, uninstall and reinstall the docker tools at the controller